CSC307Fall2018/team-too
# Add order endpoints

This pull adds `order` and `orders` queries as well as `createOrder`, `updateOrder`, and `deleteOrder` mutations.

QA
---
- Run the server locally
- Open the playground
- Assert that the queries and mutations work as expected


Closes #68
